patches/chrome-browser-media-router-BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/media/router/BUILD.gn b/chrome/browser/media/router/BUILD.gn
+index 09d54993f10174b055632b2fb5a2933a2338e8e5..5684b6ed9120484c1d6e14031631bc271f46ab23 100644
+--- a/chrome/browser/media/router/BUILD.gn
++++ b/chrome/browser/media/router/BUILD.gn
+@@ -6,6 +6,7 @@ import("//chrome/browser/media/router/features.gni")
+ import("//extensions/buildflags/buildflags.gni")
+ import("//testing/libfuzzer/fuzzer_test.gni")
+ import("//testing/test.gni")
++import("//brave/test/exclusions.gni")
+
+ static_library("router") {
+   deps = [
+@@ -265,6 +266,7 @@ source_set("unittests") {
+       "providers/openscreen/network_service_quic_packet_writer_unittest.cc",
+     ]
+   }
++  sources -= brave_chromium_unit_test_exclusions_media_router
```

Comment:
I don't see where these are defined, but can we just remove them by using chromium_src overrides to make the file empty?

Reply:
> I don't see where these are defined

GitHub is not very nice collapsing the content of the newly added `unit_tests_exclusions.gni` file, which I think is why you might have overlooked its definition, but it's indeed there. See https://github.com/brave/brave-core/pull/7901/files#diff-663d5215178a6aab5144409eee430f87810538a46612375d5e702fb0af75a8d6R276

> but can we just remove them by using chromium_src overrides to make the file empty?

We could do that, but before I do so please be aware that there are **many** `*_unittest.cc` files that are being disabled via definitions like this one from [`unit_tests_exclusions.gni`](https://github.com/brave/brave-core/pull/7901/files#diff-663d5215178a6aab5144409eee430f87810538a46612375d5e702fb0af75a8d6), so there would be quite a bunch of overrides to be made.

In the other hand, emptying those `*_unittests.cc` files would prevent us from having to maintain the tedious collection of `if`'s I had to use in [`unit_tests_exclusions.gni`](https://github.com/brave/brave-core/pull/7901/files#diff-663d5215178a6aab5144409eee430f87810538a46612375d5e702fb0af75a8d6) to make sure that we only remove a test that has been previously added according to whatever build options we're using, so it's certainly worth a go.

Will try this out now and update the PR accordingly once it's ready for another review.

Thanks!

Reply:
@bridiver I haven't yet finished doing the change for browser tests (I'm getting some strange failures there I still need to figure out) but this one ready for another look already when you have a moment. Thanks!